backend/baseclass.pm

```diff
@@ -406,16 +407,15 @@ sub enqueue_screenshot {
     # written out unconditionally (see write_img)
     if ($sim <= 54) {
         $self->last_image($image);
-        $self->_last_screenshot_name($filename);
     }
 
     if ($sim > 50) {    # we ignore smaller differences
         $self->write_encoder_frame('R');
         $watch->lap("write_encoder_frame R");
     }
     else {
-        $self->write_img($image, $filename) || die "write $filename";
-        $self->write_encoder_frame("E $filename");
+        #$self->write_img($image, $filename) || die "write $filename";
+        #$self->write_encoder_frame("E $filename");
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit